src/mediarithmics/plugins/common/BasePlugin.ts

```diff
@@ -273,7 +290,8 @@ export abstract class BasePlugin {
                              body?: any,
                              qs?: any,
                              isJson?: boolean,
-                             isBinary?: boolean): Promise<any> {
+                             isBinary?: boolean,
+                             headers?: any): Promise<any> {
```

Comment:
I'm not sure of what this code is needed for.

Did I miss something?

Reply:
@PoulpiFr You want to send the If-Modified-Since header in your request